src/components/tabs/Tabs.unit.js

```diff
@@ -15,7 +15,7 @@ describe('Tabs Component', () => {
       form.checkValidity(data, true, data);
 
       setTimeout(() => {
-        const openModalWrapper = form.element.querySelector('[ref="openModalWrapper"]');
+        const openModalWrapper = form.element.querySelector('[aria-label="modal-wrapper"]');
```

Comment:
why did we get rid of "refs" here?

Reply:
That's because I've changed the logic in a case when we have a parent component as a modal. I added the component's key of a modal's ref. In the test, I was not sure about the exact 'key' and just added as a selector 'aria-label'